server/cp/macros/desk_default_template_metadata.py

```diff
@@ -0,0 +1,92 @@
+import logging
+from flask_babel import lazy_gettext
+from superdesk import get_resource_service
+
+
+logger = logging.getLogger(__name__)
+FIELDS_TO_OVERRIDE = ['urgency']
+FIELDS_TO_EXCLUDE = ['subject',
+                     'headline',
+                     'body_html',
+                     'abstract',
+                     'byline',
+                     'dateline',
+                     'body_footer',
+                     'sign_off',
+                     'usageterms']
+
+
+def get_default_content_template(item, **kwargs):
+    if 'dest_desk_id' in kwargs:
+        desk = None
+        desk_id = kwargs['dest_desk_id']
+    elif 'desk' in kwargs:
+        desk = kwargs['desk']
+        desk_id = kwargs['desk']['_id']
+    elif 'task' in item and 'desk' in item['task']:
+        desk = None
+        desk_id = item['task'].get('desk')
+    else:
+        logger.warning("Can't set default data, no desk identifier found")
+        return
+
+    if desk is None:
+        desk = get_resource_service('desks').find_one(req=None, _id=desk_id)
+    if not desk:
+        logger.warning('Can\'t find desk with id "{desk_id}"'.format(desk_id=desk_id))
+        return
+
+    content_template_id = desk.get("default_content_template")
+    if not content_template_id:
+        logger.warning("No default content template set for {desk_name}".format(
+            desk_name=desk.get('name', desk_id)))
+        return
+    content_template = get_resource_service("content_templates").find_one(req=None, _id=content_template_id)
+    if not content_template:
+        logger.warning('Can\'t find content_template with id "{content_template_id}"'.format(
+            content_template_id=content_template_id))
+        return
+
+    return content_template
+
+
+def set_default_template_metadata(item, **kwargs):
```

Comment:
would be good to have most of it in core and just define the fields in each repository, maybe via settings, otherwise
it might be hard to fix bugs if we copy this over